_posts/0000-01-02-naveennanda.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
To remain an active panel member, we recommend that you participate frequently in surveys that are sent to you. Participation is voluntary and if you decide not to participate in a survey simply click on the decline link.

Reply:
verifed and agreed